Dockerfile

```diff
@@ -34,8 +32,8 @@ USER zulip
 WORKDIR /home/zulip/zulip
 
 # Finally, we provision the development environment and build a release tarball
-RUN ./tools/provision --production-travis
-RUN /bin/bash -c "source /srv/zulip-py3-venv/bin/activate && ./tools/build-release-tarball docker" && \
+RUN ./tools/provision --production-travis && \
+    /bin/bash -c "source /srv/zulip-py3-venv/bin/activate && ./tools/build-release-tarball docker" && \
     mv /tmp/tmp.*/zulip-server-docker.tar.gz /tmp/zulip-server-docker.tar.gz
```

Comment:
This is a 2-stage build and the whole image from here on up is discarded (see the next line).  It's purpose is a multi-stage build, and the `RUN` breaks are there to make iterative debugging efficient.  So I think these changes are not an improvement.